developers/weaviate/api/graphql/get.md

```diff
@@ -40,6 +40,190 @@ The above query will result in something like the following:
+      limit: 100
+      nearObject:{
+        id: "00000000-0000-0000-0000-000000000001"
+      }
+      groupBy:{
+        path:["content"]
+        groups:2
+        objectsPerGroup:2
+      }
+    ){
+      _additional{
+        id
+        group{
+          id
+          count
+          groupValue
+          maxDistance
+          minDistance
+          hits{
+            content
+            ofDocument {
+              ... on Document{
+                _additional{
+                  id
+                }
+              }
+            }
+            _additional{
+              id
+              distance
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+</details>
+
+Will result in the following response:
+
+<details>
+  <summary>Corresponding response</summary>
+
+```json
+{
+  "data": {
+    "Get": {
+      "Passage": [
+        {
+          "_additional": {
+            "group": {
+              "count": 1,
+              "groupValue": "Content of passage 1",
+              "hits": [
+                {
+                  "_additional": {
+                    "distance": 0,
+                    "id": "00000000-0000-0000-0000-000000000001"
+                  },
+                  "content": "Content of passage 1",
+                  "ofDocument": [
+                    {
+                      "_additional": {
+                        "id": "00000000-0000-0000-0000-000000000011"
+                      }
+                    }
+                  ]
+                }
+              ],
+              "id": 0,
+              "maxDistance": 0,
+              "minDistance": 0
+            },
+            "id": "00000000-0000-0000-0000-000000000001"
+          }
+        },
+        {
+          "_additional": {
+            "group": {
+              "count": 1,
+              "groupValue": "Content of passage 2",
+              "hits": [
+                {
+                  "_additional": {
+                    "distance": 0.00078231096,
+                    "id": "00000000-0000-0000-0000-000000000002"
+                  },
+                  "content": "Content of passage 2",
+                  "ofDocument": [
+                    {
+                      "_additional": {
+                        "id": "00000000-0000-0000-0000-000000000011"
+                      }
+                    }
+                  ]
+                }
+              ],
+              "id": 1,
+              "maxDistance": 0.00078231096,
+              "minDistance": 0.00078231096
+            },
+            "id": "00000000-0000-0000-0000-000000000002"
+          }
+        }
+      ]
+    }
+  }
+}
+```
+
+</details>
+
+:::note Performance vs `path` complexity
+Keep in mind that specifying a `path` value that requires resolving a large number of objects may be computationally expensive. For instance, setting the `path` above to `["ofDocument", "Document", "title"]` would require resolving all documents and may take considerably longer.
```

Comment:
nested cross-refs paths are not supported in the first version of `groupBy`, you can only define `["ofDocument"]` is possible for now